quodlibet/quodlibet/ext/events/lyricswindow.py

```diff
@@ -0,0 +1,368 @@
+                    window.set_zoom_level(value)
+
+            zoom_level.connect('value-changed', change_zoom_level)
+            l1 = ConfigLabel(_("_Zoom level:"), zoom_level)
+            hb.pack_start(l1, False, True, 0)
+            hb.pack_start(zoom_level, False, True, 0)
+            vb2.pack_start(hb, False, True, 0)
+            return vb2
+
+        def build_alternate_search_widget():
+            vb2 = Gtk.VBox(spacing=3)
+
+            hb = Gtk.HBox(spacing=6)
+
+            def on_entry_changed(entry, *args):
+                self.Conf.alternate_search_url = entry.get_text()
+
+            URL_entry = UndoEntry()
+            URL_entry.set_text(self.Conf.alternate_search_url)
+            URL_entry.connect("changed", on_entry_changed)
+
+            l1 = ConfigLabel(_("URL:"), URL_entry)
+
+            URL_revert = Gtk.Button()
+            URL_revert.add(Gtk.Image.new_from_icon_name(
+                Icons.DOCUMENT_REVERT, Gtk.IconSize.MENU))
+            URL_revert.set_tooltip_text(_("Revert to default"))
+
+            connect_obj(URL_revert, "clicked", URL_entry.set_text,
+                DEFAULT_ALTERNATE_SEARCH_URL)
+
+            hb.pack_start(l1, False, True, 0)
+            hb.pack_start(URL_entry, True, True, 0)
+            hb.pack_start(URL_revert, False, True, 0)
+
+            vb2.pack_start(hb, False, True, 0)
+
+            def on_alternate_search_toggled(button, *args):
+                self.Conf.alternate_search_enabled = button.get_active()
+
+            alternate_search_enabled = Gtk.CheckButton(
+                label=_("Search via above URL if the lyrics "
+                        "couldn't be found in LyricsWikia."),
+                use_underline=True)
+            alternate_search_enabled.set_active(
+                self.Conf.alternate_search_enabled)
+            alternate_search_enabled.connect("toggled",
+                on_alternate_search_toggled)
+
+            vb2.pack_start(alternate_search_enabled, False, True, 0)
+
+            return vb2
+
+        frame = qltk.Frame(label=_("Display"),
+            child=build_display_widget())
+        frame.set_border_width(6)
+        self.pack_start(frame, False, True, 0)
+
+        frame = qltk.Frame(label=_("Alternate search"),
+            child=build_alternate_search_widget())
+        frame.set_border_width(6)
+        self.pack_start(frame, False, True, 0)
+
+
+class LyricsWindow(EventPlugin):
+    PLUGIN_ID = 'lyricswindow'
+    PLUGIN_NAME = _('Lyrics Window')
+    PLUGIN_DESC = _("Shows a window containing lyrics of the playing song.")
+    PLUGIN_ICON = Icons.EDIT_FIND
+
+    _window = None
+
+    def _destroy(self, *args):
+        self._window = None
+
+    def enabled(self):
+        if self._window is None:
+            self._open_webview_window()
+
+    def disabled(self):
+        if self._window is not None:
+            self._window.close()
+
+    @cached_property
+    def Conf(self):
+        return get_config(self.PLUGIN_ID)
```

Comment:
This now seems very similar to what `PluginConfigMixin` does, essentially (but less configurable for the config section actually)...